either/README.md

```diff
@@ -477,6 +477,12 @@ const { value } = left<Error, number>(new Error()); // number | Error
 const { value } = left(new Error()); // Error
 ```
 
+```typescript
+right(2).unwrap() // number
```

Comment:
Could you please also add some interface to `"@sweet-monads/interfaces"` package like `Unwrapable`?